docs/jmd/constructors.jmd

```diff
@@ -107,38 +162,38 @@ zerocorr!
 ```
 
 The special syntax `zerocorr` can be applied to individual random effects terms inside the `@formula`:
-```@docs
-zerocorr
+```@example Main
+fm2zerocorr_fm = fit(MixedModel, @formula(reaction ~ 1 + days + zerocorr(1 + days|subj)), sleepstudy)
 ```
-```{julia;term=true}
-fit(MixedModel, @formula(reaction ~ 1 + days + zerocorr(1+days|subj)), sleepstudy)
+```@setup Main
+    all(fm2zerocorr == fm2zerocorr_fm
```

Comment:
Should there be a closing parenthesis here?

Reply:
An eager push... I was going to be testing the model with `==` on every property absent a 
`==(::LinearMixedModel, ::LinearMixedModel)`